src/server/sched_func.c

```diff
@@ -151,11 +148,9 @@ sched_free(pbs_sched *psched)
  * @retval	0	- scheduler purged
  * @retval	PBSE_OBJBUSY	- scheduler deletion not allowed
  */
-int
-sched_delete(pbs_sched *psched)
-{
-	pbs_db_obj_info_t   obj;
-	pbs_db_sched_info_t   dbsched;
+int sched_delete(pbs_sched *psched) {
```

Comment:
same as above, the function brace needs to be on its own line (same for the rest of the file).